src/SDKs/Compute/Compute.Tests/ContainerServiceTests/ContainerServiceUpdateTests.cs

```diff
@@ -57,7 +57,11 @@ public void TestContainerServiceUpdateOperations()
                     ValidateContainerService(inputContainerService, containerService);
 
                     var listResult = m_CrpClient.ContainerServices.ListByResourceGroup(rgName);
+#if NET46
```

Comment:
As we do not support .NET 46, this piece of code will never be executed.
Any reason for adding .NET 46 moniker?